tests/run-in-kind/req_crds/clusteroperators-crd.yaml

```diff
@@ -0,0 +1,280 @@
+                items:
+                  description: ObjectReference contains enough information to let
+                    you inspect or modify the referred object.
+                  properties:
+                    group:
+                      description: group of the referent.
+                      type: string
+                    name:
+                      description: name of the referent.
+                      type: string
+                    namespace:
+                      description: namespace of the referent.
+                      type: string
+                    resource:
+                      description: resource of the referent.
+                      type: string
+                  required:
+                  - group
+                  - name
+                  - resource
+                  type: object
+                type: array
+              versions:
+                description: versions is a slice of operator and operand version tuples.  Operators
+                  which manage multiple operands will have multiple operand entries
+                  in the array.  Available operators must report the version of the
+                  operator itself with the name "operator". An operator reports a
+                  new "operator" version when it has rolled out the new version to
+                  all of its operands.
+                items:
+                  properties:
+                    name:
+                      description: name is the name of the particular operand this
+                        version is for.  It usually matches container images, not
+                        operators.
+                      type: string
+                    version:
+                      description: version indicates which version of a particular
+                        operand is currently being managed.  It must always match
+                        the Available operand.  If 1.0.0 is Available, then this must
+                        indicate 1.0.0 even if the operator is trying to rollout 1.1.0
+                      type: string
+                  required:
+                  - name
+                  - version
+                  type: object
+                type: array
+            type: object
+        required:
+        - spec
+        type: object
+    served: true
+    storage: true
+    subresources:
+      status: {}
+status:
+  acceptedNames:
+    kind: ClusterOperator
+    listKind: ClusterOperatorList
+    plural: clusteroperators
+    shortNames:
+    - co
+    singular: clusteroperator
+  conditions:
+  - lastTransitionTime: "2021-08-19T07:47:23Z"
+    message: no conflicts found
+    reason: NoConflicts
+    status: "True"
+    type: NamesAccepted
+  - lastTransitionTime: "2021-08-19T07:47:23Z"
+    message: the initial names have been accepted
+    reason: InitialNamesAccepted
+    status: "True"
+    type: Established
+  storedVersions:
+  - v1
```

Comment:
please remove status and other unused fields.

Reply:
thanks, let me remove it.